# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../grsecurity/gradm-Makefile-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2005-03-16 18:25:28.429136424 +0000
+++ ./Makefile	2005-03-16 18:36:40.917902592 +0000
@@ -106,20 +106,9 @@
 	@if [ ! -f $(DESTDIR)$(GRSEC_DIR)/learn_config ] ; then \
 		$(INSTALL) -m 0600 learn_config $(DESTDIR)$(GRSEC_DIR) ; \
 	fi
-	@if [ -z "`cut -d" " -f3 /proc/mounts | grep "^devfs"`" ] ; then \
-		rm -f $(DESTDIR)/dev/grsec ; \
-		if [ ! -e $(DESTDIR)/dev/grsec ] ; then \
-			mkdir -p $(DESTDIR)/dev ; \
-			$(MKNOD) -m 0622 $(DESTDIR)/dev/grsec c 1 12 ; \
-		fi \
-	fi
+	
 	mkdir -p $(DESTDIR)$(MANDIR)/man8
 	$(INSTALL) -m 0644 gradm.8 $(DESTDIR)$(MANDIR)/man8/$(GRADM_BIN).8
-	@if [ -x /sbin/$(GRADM_BIN) ] ; then \
-		if [ -z $(DESTDIR) ] && [ ! -f $(GRSEC_DIR)/pw ] ; then \
-			/sbin/$(GRADM_BIN) -P ; \
-		fi \
-	fi
 	@true
 
 clean:
